packages/components/src/Link/Link.tsx

```diff
@@ -25,18 +24,15 @@ export type LinkProps = LinkHTMLElementProps & {
  * however, it can be styled to look like a button. In terms of the look and feel of the
  * component in the UI, the Link and Button components are exactly the same.
  *
- * Note: when using a routing component like react-router, you'll probably want to pass
- * in its Link component via the `as` prop.
+ * Note: when using a routing component like react-router, you'll probably want to
+ * use Clickable and pass in the routing component via the `as` prop.
```

Comment:
Note to self: this line will need to be updated if https://github.com/chanzuckerberg/edu-design-system/pull/627 merges first.